README.md

```diff
@@ -18,13 +23,20 @@ Parchment is an open community-sourced modloader-neutral set of mappings of para
 the official names released by Mojang. This repository contains the files which make up the Parchment mapping set, with
 the build tool to manage the mappings and generate exports for publishing.
 
-### Want to use Parchment in your workspace? See the [Getting Started](https://github.com/ParchmentMC/Parchment/wiki/Getting-Started) page in the repository wiki.
+## Usage
+To use Parchment in your workspace please see the [Getting Started](https://github.com/ParchmentMC/Parchment/wiki/Getting-Started) page in the repository wiki.
 
 ## Contributing
 See the [`docs/CONTRIBUTING.md` file](docs/CONTRIBUTING.md) for information on contributing. Also look at the 
 [repository wiki](https://github.com/ParchmentMC/Parchment/wiki) for more information, such as links to the mapping 
 standards and PR review process documents.
 
+__**Contributors:**__ 
+
+<a href="https://github.com/parchmentmc/parchment/graphs/contributors">
+  <img src="https://contrib.rocks/image?repo=parchmentmc/parchment" />
+</a>
+
```

Comment:
I don't support having this in the Readme.md.

At least not added via PR without discussing this first with the team.

Please remove this.

Reply:
Removed